cmd/deploy.go

```diff
@@ -73,7 +73,42 @@ The command performs the following steps:
 			namespace := config.namespace
 			configFile := filepath.Join(projectDir, config.appDeployFile)
 
+			exists, err := Exists(configFile)
+			if err != nil && !strings.Contains(err.Error(), "no such file") {
```

Comment:
is the string contains "no such file" necessary? the Exists() func already checks for `os.IsNotExist`

Reply:
Yup, I was getting an err directly from the Exists when trying to look for the file that is why I added that check.. but I checked and it works without it